create : 워크샵 생성, 수정, 삭제 기능

jihyun-cho-0 opened the pull request:
### PR 타입(하나 이상의 PR 타입을 선택해주세요)
- [ ] 디자인 변경
- [x]  기능 추가
- [ ] 기능 삭제
- [x] 버그 수정
- [ ] 의존성, 환경 변수, 빌드 관련 코드 업데이트
### 반영 브랜치
- jihyun -> develop
### 변경 사항
- 워크샵 시리얼라이저 수정(pk값 대신 해당 str값으로 출력되도록 수정)
- 워크샵 생성, 수정, 삭제 기능 생성
### 테스트 결과
- 워크샵 전체 목록 조회
![](https://velog.velcdn.com/images/def3ff/post/9ee4e632-a54a-4e49-a029-8674d5b2e31b/image.png)

- 특정 워크샵 상세 조회
![](https://velog.velcdn.com/images/def3ff/post/445dd79c-007a-4cb9-8a0a-3eec126b83b6/image.png)

jihyun-cho-0 committed:
create : 워크샵 생성, 수정, 삭제 기능

- 워크샵 시리얼라이저 수정(pk값 대신 해당 str값으로 출력되도록 수정)
- 워크샵 생성, 수정, 삭제 기능 생성

9yuhyeon reviewed (approved):
코드 잘 확인했습니다!
특별히 문제될 건 없어보입니당 👍

jihyun-cho-0 committed:
Merge branch 'develop' of https://github.com/Team-Hot6/Togeduck_Backend into jihyun

jihyun-cho-0 committed:
update : (워크샵) 전체 목록 조회, 상세 조회, 생성

- 워크샵 전체 목록 조회 serializers 수정
- 특정 워크샵 상세 조회 serializers 수정 (리뷰 추가)
- 워크샵 생성 시 사용되는 serializers 생성

jihyun-cho-0 committed:
update : 워크샵 수정(PUT) 오류 정정

- 생성된 워크샵 수정하기(PUT) 오류 정정
